sentry-samples/sentry-samples-spring-boot-jakarta/src/main/java/io/sentry/samples/spring/boot/jakarta/SentryCacheWrapper.java

```diff
@@ -0,0 +1,100 @@
+package io.sentry.samples.spring.boot.jakarta;
+
+import io.sentry.ISpan;
+import io.sentry.Sentry;
+import io.sentry.util.StringUtils;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.concurrent.Callable;
+import org.springframework.cache.Cache;
+
+public class SentryCacheWrapper implements Cache {
```

Comment:
We've just implemented some methods here to showcase how this works. The rest still needs to be implemented.